spring-cloud-dataflow-composed-task-runner/README.adoc

```diff
@@ -0,0 +1,128 @@
+`&&` symbol then the next task definition to be launched.
+For example if we have tasks AAA, BBB and CCC to be launched in sequence it
+will look  like this:
+```
+AAA && BBB && CCC
+```
+image::{image-root}/basicsequence.png[basic sequence]
+
+You can execute the same task multiple times in a sequence.  For example:
+```
+AAA && AAA && AAA
+```
+image::{image-root}/samejobsequence.png[basic sequence with repeated job definition]
+
+If an `ExitStatus` 'FAILED' is returned in a sequence the Composed Task
+Runner will terminate.  For example if `AAA && BBB && CCC` composed task is
+executed and BBB fails.  Then CCC will not be launched.
+
+=== Transitions
+The Composed Task Runner supports the ability to control what tasks get
+executed based on the `ExitStatus` of the previous task.  This is
+done by specifying `ExitStatus` after the task definition followed by
+the `->` operator and the task definition that should be launched based on
+the result.  For example:
+```
+AAA 'FAILED' -> BBB 'COMPLETED' -> CCC
+```
+image::{image-root}/basictransition.png[basic transition]
+
+Will launch AAA and if AAA fails then BBB will be launched.  Else if AAA
+completes successfully then CCC will launch.
+
+You can also have a sequence that follows a transition.  For example:
+```
+AAA 'FAILED' -> BBB && CCC && DDD
+```
+image::{image-root}/basictransitionwithsequence.png[basic transition with sequence]
+
+Will launch AAA and for any `ExitStatus` that is returned other than 'FAILED' then
+CCC && DDD  will be launched.  However if AAA returns 'FAILED' then BBB will
+be launched, but CCC && DDD will not.
+
+==== Wildcard
+Wildcards are also supported in transitions.
+For example:
+```
+AAA 'FAILED' -> BBB '*'->CCC
+```
+image::{image-root}/basictransitionwithwildcard.png[basic transition with wildcard]
+
+In the case above AAA will launch and any `ExitStatus` other than FAILED will
+launch CCC.
+
+=== Splits
+Allows a user to execute tasks in parallel.
+For example:
+```
+<AAA || BBB || CCC>
+```
+image::{image-root}/basicsplit.png[basic split]
+
+Will launch AAA, BBB and CCC in parallel.   When launching splits as a part of a
+composed task all elements of the split must finish successfully before the
+next task definition can be launched for example:
+```
+<AAA || BBB || CCC> && DDD && EEE
+```
+image::{image-root}/basicsplitwithsequence.png[basic split with sequence]
+
+In the case above once AAA, BBB and CCC complete sucessfully then DDD and EEE
+will be launched in the sequence enumerated above.  However if one of the task
+definitions fails in the split then DDD and EEE will not fire.  For example if
+BBB fails then AAA and CCC will be marked successful and BBB will be marked a
+failure and DDD and EEE will not be launched.
+
+== Configuration
+
+See the [Configuring Composed Task Runner](https://docs.spring.io/spring-cloud-dataflow/docs/current/reference/htmlsingle/#spring-cloud-dataflow-composed-tasks) in the Spring Cloud Data Flow reference guide.
```

Comment:
Update the link from markdown to asciidoc i.e. https://docs.spring.io/spring-cloud-dataflow/docs/current/reference/htmlsingle/#spring-cloud-dataflow-composed-tasks[Configuring Composed Task Runner].  Will fix on merge